pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -114,7 +121,9 @@ def remove_place(self, place):
         :param place: Place person should be removed from
         :type place: Place
         """
-        if place not in self.places:
+        place_list = [i[0] for i in self.places]
+        if place not in place_list:
             raise KeyError("Person not found in this place")
         else:
-            self.places.remove(place)
+            ind = place_list.index(place)
+            self.places.pop(ind)
```

Comment:
If all people from a group are removed from a place, will we need to reassigned the group index?

Reply:
Not sure. The way I've suggested in the PlaceType enum has each index vaguely meaning something (workers, visitors etc) but that was only a suggestion

Reply:
Ahh yes that makes sense!